### DISCUSSION TITLE
Question about simulating nucleation growth process.

### DISCUSSION THREAD

**Discussion by theAfish** [deepmodeling/deepmd-kit#980] [Q&A] [answered]
Hi all, I am a new learner and interested in using DPMD to explore the low-dimensional crystallization process in solution. There are two questions I would like to ask here, although they may not be about the DPMD itself.

1. In the growing process of crystal, the solvents may play important roles. However, for crystal systems with complex element compositions, adding solvent molecules into the simulation or training process would increase the types of atoms to be considered and the calculation time. So, is it possible, or necessary to use machine learning to fit an implicit solvent field that could be used in multiple situations? I think maybe we could use this implicit solvent field as a special thermostat for solvent systems.

2. When we are interested in the growth process of a single nucleus, if we do not want to add solvent molecules or want to study the process of gaseous crystallization, directly put the particles to be crystallized into the simulation box could cause unwanted aggregation beyond the target nucleus. So, could we add one atom or molecule waiting to crystallize at a time, and wait for it to stabilize before adding a new one. Just like the DLA (diffusion-limited Aggregation) model in a grid system? Is it possible to do this in the DPMD framework via lammps?

Thank you very much!

![image](https://user-images.githubusercontent.com/85424334/129532718-94be4558-be79-4fd9-9e7c-aec7303133d1.png)

**Comment by njzjz** [comment, marked as answer]
1. You can, but note that the accuracy may be downgraded.
2. I don't think it's related to DPMD. DPMD only provides potential energies and forces to MD simulations, and do not refer to the MD simulations itself. You are free to use any other commands in LAMMPS.

**Reply by theAfish** [reply]
Thank you very much!
